src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -553,7 +554,26 @@
     </Touch>
   </Target>
 
+  <Target Name="GetProducedPackagesFromDependentVerticals"
+          Condition="'$(IsUtilityProject)' != 'true'"
+          Returns="@(ProducedPackageFromDependentVertical)">
+    <!-- Add manifests from dependent verticals. -->
+    <ItemGroup>
+      <RepoAssetManifestFromDependentVertical Include="$(ArtifactsAssetManifestsDir)*.xml" Exclude="$(MergedAssetManifestOutputPath)" />
+    </ItemGroup>
+
+    <GetKnownArtifactsFromAssetManifests AssetManifests="@(RepoAssetManifestFromDependentVertical)" RepoOrigin="$(RepositoryName)">
+      <Output TaskParameter="KnownPackages" ItemName="ProducedPackageFromDependentVertical" />
+      <Output TaskParameter="KnownBlobs" ItemName="ProducedAssetFromDependentVertical" />
```

Comment:
I'm not sure why but `ProducedAssetFromDependentVertical` seems to be unused.

Reply:
We don't flow assets across the ProjectReference boundary, so I guess we don't currently have a use for it.